Assets/Project/Scripts/UnityChanControlScript.cs

```diff
@@ -4,12 +4,21 @@
 
 namespace UnityChan
 {
+    public class Ability
```

Comment:
Would usually say this should be in a new file but in this case, I think it's fine